File: a059.cpp
#include <iostream>
#include <cmath>
using namespace std ;

int main()
{
    int n;
    while(cin >> n)
    {
        for(int i=0;i<n;i++)
        {
            int sum=0,a,b;
            cin >> a >> b;
            for(int j=a;j<=b;j++)
            {
                for(int k=1;k<=sqrt(j);k++)
                {
                    if(j==k*k)
                        sum+=j;
                }
            }
            cout << "Case " << i+1 << ": " << sum << endl;
        }

    }
}

File: a059.py
import math
table = [0] * 1001
for i in range(1, 1001):
    if math.sqrt(i).is_integer():
        table[i] = 1
t = int(input())
for i in range(t):
    a = int(input())
    b = int(input())
    sum=0
    for j in range(a, b+1):
        if table[j] == 1:
            sum += j
    print("Case {}: {}".format(i+1, sum))
